<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectionLayerWidget</class>
 <widget class="QWidget" name="SelectionLayerWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>244</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item row="2" column="0">
    <widget class="QGroupBox" name="displayGroupBox">
     <property name="title">
      <string>Display</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="ColorPickerPushButton" name="colorPickerPushButton" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="autoZoomToSelectionCheckBox">
        <property name="text">
         <string>Auto zoom to selection</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="colorLabel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="zoomToSelectionPushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Zoom to selection</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="pixelSelectionGroupBox">
     <property name="title">
      <string>Pixel selection</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="selectionModifierHorizontalLayout">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="selectAllPushButton">
          <property name="text">
           <string>All</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectNonePushButton">
          <property name="text">
           <string>None</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="invertSelectionPushButton">
          <property name="text">
           <string>Invert</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Rectangle</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Brush</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Polygon</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Lasso</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="pixelSelectionTypeLabel">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorPickerPushButton</class>
   <extends>QWidget</extends>
   <header>colorpickerpushbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
